* Question 1: "values added: 20"
* Question 2: "final result: 20"
* Question 3: "values added:  20"
* Question 4: This gives an error saying that the variable 'result' does not exist. The reason it gives this error is because declaring a variable with the let keyword only gives it a block scope. This means that it can only be accessed within the block it is defined in. Since result is not defined in line 13's block it gives the error saying the variable result does not exist. 
* Question 5: Because the variable 'result' is definied with the const keword, it not allowed to be reassigned which is why an error is thrown and line 9 is not outputted.
* Question 6: Because the variable 'result' is definied with the const keword, it not allowed to be reassigned which is why an error is thrown and line 13 is not outputted.